pylint/test/unittest_checker_format.py

```diff
@@ -138,6 +138,20 @@ def testCheckKeywordParensHandlesUnnecessaryParens(self):
             with self.assertAddsMessages(msg):
                 self.checker._check_keyword_parentheses(_tokenize_str(code), offset)
 
+    def testCheckIfArgsAreNotUnicode(self):
+        self.checker._keywords_with_parens = set()
+        cases = [
+            (Message('superfluous-parens', line=1, args='if'),
```

Comment:
Sorry to mess once again with this change, but `Message()` instances are unused, they should be removed before merge.